usageReports/rds/es_request_constructor.go

```diff
@@ -21,6 +20,20 @@ import (
 	"gopkg.in/olivere/elastic.v5"
 )
 
+// getDateForDailyReport returns the end and the begin of the date of the report based on a date
```

Comment:
This comment is as useful in itself as the function declaration, since it gives you the same information. You should describe how the beginning and end are computed from the date passed in params.